packages/manager/src/queries/aglb/serviceTargets.ts

```diff
@@ -7,12 +7,16 @@ import {
 } from '@linode/api-v4/lib/types';
 import { useQuery } from 'react-query';
 
-export const queryKey = 'entrypoint';
+export const queryKey = 'aglbs';
```

Comment:
```suggestion
export const queryKey = 'aglb';
```
Are we intentionally using plural here to stick to convention with the other query keys? Do we also need to update the `queryKey` in `src/queries/aglb/loadbalancers.ts`?

Reply:
Yes, using `aglbs` to follow patter of `volumes`, `linodes`, etc..

I will consolidate those query keys.